src/ajax/check_hgvs_dialogue.php

```diff
@@ -575,17 +579,26 @@ function update_images_per_step($sStep, $sImage)
     }
 
 
-    // Fill in all fields that remained empty.
+    // Fill and deactivate all fields that remained empty.
+    // Some fields might not have been filled after the mapping. Some
+    //  unknown issues have occurred here, most likely concerning the
+    //  reference sequence. Because we want to allow users to fill in
+    //  these fields without that resulting in a reset of all values,
+    //  we will deactivate the onChanges for these fields and warn the
+    //  user.
     print('
     // Fill in all fields that remained empty.
-    $(\'#variantForm input[name*="VariantOnGenome"]\').each(function(e){
-        if($(this).val()==="") {
-            $(this).val("g.?");
-        }
-    });
-    $(\'#variantForm input[name*="VariantOnTranscript"]\').each(function(e){
-        if($(this).val()==="" && !$(this).attr("name").endsWith("Exon")) {
-            $(this).val("c.?");
+    $(\'#variantForm input[name*="VariantOn"]\').each(function(e){
+        sName = $(this).attr("name");
+        if ($(this).val() === ""
+            && !(sName.endsWith("Exon") || sName.endsWith("DBID") || sName.endsWith("Reference"))) {
```

Comment:
Negative selection scares me - what if other fields are created that you may not have anticipated?